pkg/virt-handler/vm.go

```diff
@@ -1072,7 +1072,7 @@ func (d *VirtualMachineController) defaultExecute(key string,
 		case domainExists:
 			// The VirtualMachineInstance is deleted on the cluster, and domain is not alive
 			// then delete the domain.
-			log.Log.Object(vmi).V(3).Info("Shutting down domain for deleted VirtualMachineInstance object.")
+			log.Log.Object(vmi).V(3).Info("Deleting domain for deleted VirtualMachineInstance object.")
```

Comment:
I think you should add `Deleting **libvirt** domain..` to emphasize it is related to libvirt

Reply:
All the other log messages regarding the domain don't specify it is libvirt domain, so I prefer tp be consistent. Another PR can be sent to update all the relevant messages.

Reply:
yes, ideally we don't talk about libvirt in the context of virt-handler.